app/src/main/res/layout/fragment_shopmanagement.xml

```diff
@@ -12,27 +12,35 @@
             />
 
     </data>
-    <RelativeLayout
+    <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
+        android:orientation="vertical"
         >
-        <android.support.v7.widget.RecyclerView
+        <RelativeLayout
```

Comment:
WARNING: [UselessParent] This `RelativeLayout` layout or its `LinearLayout` parent is useless
 A layout with children that has no siblings, is not a scrollview or a root layout, and does not have a background, can be removed and have its children moved directly into the parent for a flatter and more efficient layout hierarchy.